uaclient/cli.py

```diff
@@ -320,6 +323,20 @@ def print_version(_args=None, _cfg=None):
     print(version.get_version())
 
 
+def action_refresh(args, cfg):
+    if not cfg.is_attached:
```

Comment:
This doesn't check if we're root, but the similar code in `action_enable` does; is that an intentional difference?

Reply:
good point. that's a bug, will fix thanks.